agir/lib/autocomplete_filter.py

```diff
@@ -69,7 +69,7 @@ class AutocompleteFilter(SelectModelFilter):
 
     class Media:
         js = (
-            "admin/js/jquery.init.js",
+            "admin/js/jquery.initFromScriptTag.js",
```

Comment:
```suggestion
            "admin/js/jquery.init.js",
```
Il faut pas changer ici, non ?

Reply:
anéfé